### ISSUE TITLE
ListView/GridView extended/multiple selection behaviour is different than WinUi

### ISSUE THREAD

**Issue by manfromarce** [unoplatform/uno#10870]
### Current behavior

1. It seems that Extended SelectionMode is not supported (it behaves like SelectionMode.Multiple)
2. Checkboxes aren't displayed when SelectionMode=Multiple and the shift key doesn't allow to select/deselect more items at once
I've only tested these in WASM and Skia.GTK at the moment, so I don't know if they work properly in other platforms.
I'm currently using GridView but the issues are probably in ListView since GridView derives from it.

### Expected behavior

1. When SelectionMode is Extended, the selection should be cleared when clicking another element unless Ctrl or Shift are pressed
2. When SelectionMode is Multiple, the shift key should allow to select all items from the last selected to the currently clicked one
3. Checkboxes should be displayed in multiple selection mode (not essential)

### How to reproduce it (as minimally and precisely as possible)

1. Extended: https://playground.platform.uno/#118b1b5a
2. Multiple: https://playground.platform.uno/#7d0b60b5

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

4.6.39

### Affected platforms

WebAssembly, Skia (GTK on Linux/macOS/Windows)

### IDE

Visual Studio 2022

### IDE version

17.4.3

**Comment by ritt-app** [comment]
I think it is very important to implement Extended SelectionMode as it allows the users to quickly select many items using Shift+Click. The lack of this feature is a major roadblock preventing me from migrating Ritt from WinUI to Uno (Skia for Mac and Linux).

**Comment by ramezgerges** [comment]
This is done in https://github.com/unoplatform/uno/pull/12909.

**Comment by manfromarce** [comment]
I've tried the latest dev version and it's fixed. However, checkboxes are not displayed, is there another open issue to track this?

**Comment by jeromelaban** [comment]
@manfromarce indeed, if you're not seeing checkboxes, it's a problem. If you can create a repro an open an issue, it'll be helpful, thanks!

**Comment by manfromarce** [comment]
Yes, I don't see them and I have SelectionMode=Multiple. I will open another issue

**Comment by MartinZikmund** [comment]
@manfromarce as the new issue is opened (https://github.com/unoplatform/uno/pull/13349), closing this
